roles/openshift_logging/files/generate-jks.sh

```diff
@@ -73,6 +73,7 @@ function generate_JKS_chain() {
         -keystore $dir/$NODE_NAME.jks \
         -storepass $ks_pass \
         -noprompt \
+        -trustcacerts \
```

Comment:
@sferich888 To answer most of your questions, its because this script is essentially what the deployer was doing is a direct port from that and was reused in the interest of time and resource constraints.  I am unable to answer the question related to MW, but we have https://trello.com/c/6a9bxQdX/445-2-jks-certs-on-container-start-of-logging-logging which is to address item 2 in your question.  Ultimately we wish to take advantage of a client cert generation service...where is that for us?